examples/java/spring-boot/src/test/java/uk/gov/api/springboot/architecture/ArchitectureLayeringTest.java

```diff
@@ -87,7 +87,7 @@ void domainRingIsSelfSufficient(JavaClasses classes) {
         .resideInAPackage(DOMAIN_RING)
         .should()
         .onlyAccessClassesThat()
-        .resideInAnyPackage(DOMAIN_RING, "java..")
+        .resideInAnyPackage(DOMAIN_RING, "java..", "org.assertj..", "org.mockito..")
```

Comment:
:thinking: I'm not sure. I'd say that we should scope this differently, and make sure that `classes()` targets only production code. Test code isn't as important for what classes are used (at least right now, and in comparison with production code)